GOVERNANCE.md

```diff
@@ -1,114 +1,185 @@
+#### How to become an approver
 
-### Approver responsibilities
+* Becoming an approver is considered both a privilege and a responsibility.
+	* The barrier to become an approver is naturally higher than that of becoming a reviewer, and thus further encompasses all of the requirements of a reviewer.
+	* Approvers should serve as a reviewer for at least three months, though this alone does not necessarily qualify a reviewer to be an approver.
+	* Approvers are community members with a vested interest (which need not be professional) in the future direction of CRI-O.
+	* An approver is not just someone who can make changes, but someone who has demonstrated their ability to collaborate with the team, get the most knowledgeable people to review code and docs, contribute high-quality code, and follow through to fix issues (in code or tests).
+	* They are expected to have a deep understanding of various pieces of CRI-O and how they work together.
+	* They are expected to be able to participate in the management of the project--joining in the discussion of technical questions and aiding in resolving technical challenges.
+* The person in question for approvership can nominate themselves, or be nominated by someone else.
+	* A contributor is nominated by someone submitting a PR to add their github handle to the approvers section of the [OWNERS](OWNERS.md) file.
+* To become an approver, a simple majority vote among the current approvers should be performed.
+    * See [conflict resolution](#conflict-resolution-and-voting) for more information on how to handle disputes.
 
-* Approvers have all of the responsibilities of the reviewers, as well as:
+#### Approver responsibilities
+
+Approvers have all of the responsibilities of the reviewers, as well as:
 * The expectation of reviews is higher. Approvers should be experts of their knowledge domain, and should serve as code owners of those pieces.
 * Participate when called upon in the [security release process](https://github.com/cri-o/cri-o/blob/main/SECURITY.md). Note that although this should be a rare occurrence, if a serious vulnerability is found, the process may take up to several full days of work to implement. This reality should be taken into account when discussing time commitment obligations with employers.
 
-## Emeritus Approver
+#### Distinction between top-level approvers and sub-tree approvers
+Through the mechanism of the OWNERS file, it’s possible to define sub-tree approvers for areas of specialty. For the purposes of this document, when “approvers” is mentioned, sub-tree approvers are not automatically included, though they could be. One way this happens is if a section of code that a sub-tree approver is responsible for has a security vulnerability, they will be included in the discussions. However, they need not necessarily be included if there is not such a vulnerability.
```

Comment:
```suggestion
Through the mechanism of the OWNERS file, it’s possible to define sub-tree approvers for areas of specialty. One way this happens is if a section of code that a sub-tree approver is responsible for has a security vulnerability, they will be included in the discussions. However, they need not necessarily be included if there is no such vulnerability. For the purposes of this document, when “approvers” is mentioned, sub-tree approvers are not automatically included, though they could be.
```

Reply:
and maybe: "when "approvers" is" to "when the word "approvers" is"

Reply:
I have dropped the security section as upon reading again I found it clunky